extensions/mongoProcessor/MongoQueueProcessor.js

```diff
@@ -196,7 +204,20 @@ class MongoQueueProcessor {
         entry.setLocation(editLocations);
     }
 
-    _updateReplicationInfo(entry) {
+    /**
+     * Update replication info on ingested object MD to match Zenko defined
+     * replication info.
+     * @param {ObjectQueueEntry} entry - object queue entry object
+     * @param {BucketInfo} bucketInfo - bucket info object
+     * @return {undefined}
+     */
+    _updateReplicationInfo(entry, bucketInfo) {
+        const bucketRepInfo = bucketInfo.getReplicationConfiguration();
```

Comment:
Could be worthwhile to abstract and reuse the logic in https://github.com/scality/cloudserver/blob/development/8.1/lib/api/apiUtils/object/getReplicationInfo.js#L95.